### DISCUSSION TITLE
GenBank nt database

### DISCUSSION THREAD

**Discussion by marchoeppner** [bio-raum/FooDMe2#11] [General]
@gregdenay  So I am giving in and will add GenBank nt - but I noticed that your script from FoodMe installs the "full" GenBank nt database. Is this by design? There is a second version, only for eukaryotes - which seems to be a bit smaller at least. Any reason not to use that?

https://ftp.ncbi.nlm.nih.gov/blast/db/

**Comment by gregdenay** [comment]
I am not sure it existed 2 years ago. It is a good idea as it is "only" 250GB. 
Ideally we would only need the organelles genomes but it doesn't seem to exist as such
